src/dmd/mars.d

```diff
@@ -1387,6 +1387,8 @@ void addDefaultVersionIdentifiers()
         VersionCondition.addPredefinedGlobalIdent("assert");
     if (global.params.useArrayBounds == CHECKENABLE.off)
         VersionCondition.addPredefinedGlobalIdent("D_NoBoundsChecks");
+    else // Introduced in 2.082.0 - Deprecate `D_NoBoundsChecks` later
```

Comment:
Use the deprecation comment syntax proposed in https://github.com/dlang/DIPs/blob/master/DIPs/DIP1013.md for easy grepping, and remain consistence with other deprecations in the code base.